src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
-public class Duke {
+import java.util.Scanner;
+import java.util.Random;
+// import java.util.Arrays;
+
+public class Duke extends DukeCommands{
     public static void main(String[] args) {
+        // Boot sequence
+        Random rand = new Random();
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello from\n" + logo + spacer);
+        System.out.println(" Hello! I'm Duke\n" + " What can I do for you?\n" + spacer);
+        Scanner sc = new Scanner(System.in);
+
+        String word = sc.nextLine();
```

Comment:
Perhaps a different variable name might be better as you are reading in one whole line and not a single word